Runar/Runar/Controllers/Generator flow/SelectRune/SelectRuneCell.swift

```diff
@@ -42,12 +42,15 @@ class SelectRuneCell: UICollectionViewCell {
             runeImage.centerYAnchor.constraint(equalTo: centerYAnchor)
         ])
     }
-    
-    public func setRune(_ rune: GenerationRuneModel, _ indexPath: IndexPath) {
+
+    public func setRune(_ rune: GeneratorCoreData, _ indexPath: IndexPath) {
```

Comment:
@Qeezii how GeneratorCoreData matches with rune?

Reply:
GeneratorCoreData is a model for runes